assets/js/modules/analytics/components/common/AcquisitionPieChart.js

```diff
@@ -66,11 +66,10 @@ function AcquisitionPieChart( { data, args, source } ) {
 
 	let sourceMessage = '';
 	if ( source ) {
-		sourceMessage = sprintf(
-			/* translators: %1$s: URL to Analytics Module page in Site Kit Admin, %2$s: Analytics (Service Name) */
-			__( 'Source: <a class="googlesitekit-cta-link googlesitekit-cta-link--external googlesitekit-cta-link--inherit" href="%1$s">%2$s</a>', 'google-site-kit' ),
+		sourceMessage = __( 'Source:', 'google-site-kit' ) + sprintf(
+			' <a class="googlesitekit-cta-link googlesitekit-cta-link--external googlesitekit-cta-link--inherit" href="%1$s">%2$s</a>',
```

Comment:
Here is another concatenation which should be updated. Even though `Source:` and the link are essentially separate, it isn't RTL friendly.

This would be another good use for `createInterpolateElement` I think so that we wouldn't need to include all of the classnames in the translation string (or extract them to a placeholder). That would allow us to use the `Link` component as well, rather than using its resulting HTML as a string which will be a pain to maintain if we were to make changes to the base `Link` component.